marxs/optics/polarization.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+
+from ..math.pluecker import *
+
+
+def polarization_vectors(dir_array, angles):
+	n = len(angles)
+	polarization = np.zeros((n, 4))
+	x = np.array([1., 0., 0.])
+	y = np.array([0., 1., 0.])
+	for i in range(0, n):
```

Comment:
I think I fixed this but it is pretty complicated (I used a lot of `np.tile` and transpose) so I am leaving the old code in commented. That way we can discuss it tomorrow and compare the versions without looking through commits.